docs/wsrep-system-index.md

```diff
@@ -1240,6 +1240,42 @@ As of *Percona XtraDB Cluster* 8.0.26-16, this variable is deprecated and may be
 Defines whether unique key checking is done for applier threads.
 This is disabled by default.
 
+### `wsrep_SR_store`
+
+| Option         | Description        |
+| -------------- | ------------------ |
+| Command Line:  | ``--wsrep-sr-store`` |
+| Config File:   | Yes                |
+| Scope:         | Global            |
+| Dynamic:       | No                 |
+| Default Value: | ``table`` |
+
+Defines storage for streaming replication fragments. The default value is `table`. To disable this variable, specify `none`.
+
+### `wsrep_sst_allowed_methods`
+
+| Option         | Description        |
+| -------------- | ------------------ |
+| Command Line:  | ``--wsrep_sst_allowed_methods`` |
+| Config File:   | Yes                |
+| Scope:         | Global            |
+| Dynamic:       | No                 |
+| Default Value: | ``IN_FS_CHARSET`` |
+
+Percona XtraDB Cluster 8.0.20-11.3 adds this variable.
+
+The variable specifies which SST methods are accepted by the server. To specify multiple SST methods, separate them by commas. The default value is `IN_FS_CHARSET`.
```

Comment:
Lots of "specifies"
Maybe 
To select multiple SST methods

Reply:
updated